src/components/EventMap.js

```diff
@@ -61,50 +65,60 @@ export default function(store){
     },
     methods: {
       plotEvents() {
-        // wipe out existing plotted events -- probably this can
-        // be done more efficiently
-        if (this.eventsLayer) {
-          this.eventsLayer.clearLayers();
-        }
-
-        this.eventsLayer = L.layerGroup().addTo(this.mapRef);
+        const eventsSource = this.mapRef.getSource("events");
 
-        this.filteredEvents.forEach((event) => {
-          L.circleMarker([event.lat, event.lng], {
-            radius: 5,
-            color: 'white',
-            fillColor: '#ef3030',
-            opacity: 0.8,
-            fillOpacity: 0.7,
-            weight: 2
-          }).addTo(this.eventsLayer);
-        });
+        if (eventsSource) {
```

Comment:
I feel like might be worth a comment to note when eventsSource does _not_ exist -- I'm not positive but it seems like this could happen if the events data loaded before the map finished loading and `mapMounted` is called